### ISSUE TITLE
Elementor Panel Difficult to Use

### ISSUE THREAD

**Issue by Mij-Strebor** [elementor/elementor#13131]
I find the scrolling of the Elementor Panel very difficult to use.  Here are the problems I'm having:

1. Scroll bar is too narrow. Very difficult to grab.
2. I haven't found a way to scroll the panel with the keyboard.
3. The scrollbar snaps to beginning or end; no smooth movement up or down.
4. If the scrollbar snaps then the selection pop-up (i.e. Color Picker, Typography, etc.) closes.

I'd like to see a smooth scrolling, normal width (Edit Panel size) scrollbar.  If I'm dragging the scroll up or down, it doesn't snap to the beginning or end; but if I click on a vacant spot on the scrollbar, the positioning will snap to cover that point.  If there isn't a keyboard method for moving the position, please use the up/down arrow keys.  Then don't close the option pop-ups automatically for any reason.

I love Elementor but get so frustrated by this panel, I mumble about seeing what competiors do. (Sometimes...)

**Comment by DorShahar** [comment]
Hi @Mij-Strebor 
Thank you for your input.
Can you please share more light about the problem?

1. The left panel is adjustable, you can widen or narrow it. It will help us to improve the UX if you can add a video of you trying to change it.  
2. Accessibility is in our area of focus. Thanks for pointing out the problem.
3. Please add a short video about your points number 3 & 4 and system information to help us to understand if there is a bug.

Many thanks.

**Comment by Mij-Strebor** [comment]
Hi, Dor. Here is a screenshot of the first point.
[image: image.png]

Problem 3: *T**he scrollbar snaps to beginning or end; no smooth movement
up or down.  **A mistake on my part! *I clicked on a blank part of the
scrollbar and it snapped to that location. The bar was long enough to look
like it jumped all the way to the beginning or end.  This snap feature is
exactly what I wanted to suggest.  Sorry.

Problem 4: *If the scrollbar snaps then the selection pop-up (i.e. Color
Picker, Typography, etc.) closes.*  I'll have to install video capture.
Here's step by step; let me know if it works and you can see the problem:

   1. Add a Section and add Posts to it.
   2. Go to Style and open Box Shadow - this should be far enough down the
   panel so that most of the pop-up is hidden.
   3. Go to the scrollbar and click in the white part indicating
   "not-visible."

The scrollbar will snap to the bottom of the panel, but the Box Shadow
pop-up will close.

Note that if I grab the scrollbar and move it down -- not using click, but
drag -- the pop-up for Box Shadow will stay open.  I suggest they should
act the same.

**Comment by Mij-Strebor** [comment]
![image](https://user-images.githubusercontent.com/17260842/100001881-030ac280-2d81-11eb-819e-e97c3758d8fa.png)

Here's the image that should have been in my reply.

**Comment by drawcard** [comment]
@Mij-Strebor is your operating system macOS? It looks like the system scrollbar, in which case that wouldn't be something controllable by Elementor:

![image](https://user-images.githubusercontent.com/1435638/100053596-498b1a80-2e70-11eb-88f7-cf428edb8949.png)

**Comment by DorShahar** [comment]
Hi @Mij-Strebor and @drawcard 
We used a library for the scroll bar. You can go to this file: `elementor/assets/dev/scss/editor/_perfect-scrollbar.scss` and change the CSS as you wish.

As for your no.4 problem - you are right about the behavior. This is since you are changing your focus by clicking the mouse outside the box-shadow popup.
With that being said - you can use the mouse wheel (or touchpad) to scroll, and it will not snap (since you don't change the focus).

So it isn't a buy - it is the normal behavior. Closing the issue.

All the best.

**Comment by ygirod** [comment]
> ![image](https://user-images.githubusercontent.com/17260842/100001881-030ac280-2d81-11eb-819e-e97c3758d8fa.png)
> 
> Here's the image that should have been in my reply.

Go to wp-content/plugins/elementor/assets/css/editor.min.css and change:

1. .ps__thumb-y{width:6px;}
2. .ps__rail-y:focus>.ps__thumb-y,.ps__rail-y:hover>.ps__thumb-y{width:6px;}
